build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:2.3.1'
         classpath 'com.google.gms:google-services:3.1.0'
-        classpath "io.realm:realm-gradle-plugin:3.3.0"
+        classpath "io.realm:realm-gradle-plugin:3.3.1"
```

Comment:
remove unrelated changes.
Realm updates often so no need of staying ahead with each update.
Let us keep a target to update it when ver 3.5 is released